syft/execution/plan.py

```diff
@@ -113,6 +107,7 @@ def __init__(
         id: Union[str, int] = None,
         owner: "sy.workers.BaseWorker" = None,
         tags: List[str] = None,
+        serialized_input: list = None,
```

Comment:
Could we call this field something like `input_types`? I've been finding `serialized_input` a bit confusing.